python/src/aac/execute/plugin_manager.py

```diff
@@ -0,0 +1,46 @@
+"""Provide access to plugins and plugin data."""
+
+from importlib import import_module
+from pkgutil import iter_modules
+from types import ModuleType
+from pluggy import PluginManager
+from typing import List
+
+from aac.execute import PLUGIN_PROJECT_NAME, hookspecs
+
+
+def get_plugin_manager() -> PluginManager:
+    """
+    Get the plugin manager and automatically registers first-party internal plugins.
```

Comment:
If we no longer have the concept of first and third party, believe this is the more accurate phrasing.

```suggestion
    Get the plugin manager and automatically register core plugins.
```